### DISCUSSION TITLE
Add ability to connect to two servers

### DISCUSSION THREAD

**Discussion by daltay15** [Radiokot/photoprism-android-client#142] [Q&A] [answered]
I would love to see the ability to connect to two different server addresses, one on the LAN and Another for when I'm not on the LAN. Currently I have to use the Web app when I'm not home but it'd be great to use one app for all.

**Comment by Radiokot** [comment, marked as answer]
Hello Demir.
I don't want to add such complexity to the app. But there's a solution – use the clone package to simultaneously connect to two libraries. In your app menu you'll have two PhotoPrism Gallery instances. Each [release on GitHub](https://github.com/Radiokot/photoprism-android-client/releases/tag/1.27.0(41)) has both main and clone APK attached.

**Comment by daltay15** [comment]
I completely get that, that solution would work perfectly, thank you!!
